spec/models/user_spec.rb

```diff
@@ -103,4 +103,35 @@
       it { expect { subject }.to change(child2, :organisation_ids).from([organisation.id]).to([]) }
     end
   end
+
+  describe "#soft_delete" do
+    let(:now) { Time.current }
+    before do
+      freeze_time
+      user.soft_delete(deleted_org)
+    end
+
+    context 'belongs to multiple organisations and with organisation given' do
+      let(:user) { create(:user, :with_multiple_organisations) }
+      let(:deleted_org) { user.organisations.first }
+      it { expect(user.organisation_ids).not_to include(deleted_org.id) }
+      it { expect(user.organisation_ids).not_to be_empty }
```

Comment:
Tu peux remplacer ça par un `expect(user.organisation_ids.size).to eq(2)` pour être en plus précis.
Voir si t'arrive à savoir les ids qui doivent être présent `to contain_exactly(id1, id2)`